collect_app/src/test/java/org/odk/collect/android/location/activities/BaseGeoActivityTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.location.activities;
+
+import org.junit.Before;
+import org.robolectric.shadows.ShadowApplication;
+
+public abstract class BaseGeoActivityTest {
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
Should this be annotated with `@Before` as it gets called from the child classes

Reply:
Right, fixed.